Code/Source/RobotImporter/Pages/FileSelectionPage.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "FileSelectionPage.h"
 #include <AzCore/Utils/Utils.h>
+#include <QVBoxLayout>
 
```

Comment:
QHBoxLayout is also used in the code below. Perhaps it is included somewhere else include, but best to include it as well (add).